pom.xml

```diff
@@ -138,6 +138,7 @@
         <module>debezium-scripting</module>
         <module>debezium-server</module>
         <module>debezium-testing</module>
+        <module>connect-rest-extension</module>
```

Comment:
@jpechane, do we need any special config for getting this published as a JAR to Maven Central?